packages/cli/README.md

```diff
@@ -36,7 +36,7 @@ $ npm install -g @blitzjs/cli
 $ blitz COMMAND
 running command...
 $ blitz (-v|--version|version)
-@blitzjs/cli/0.0.1-canary.1 darwin-x64 node-v12.16.1
+@blitzjs/cli/0.0.1-canary.1 darwin-x64 node-v13.9.0
```

Comment:
This stuff is noise too.

Reply:
@merelinguist is your editor set up to use the `prettier.config.js` in the project root? That should keep this from happening.

Also, the husky pre-commit hook should be running to format it according to `prettier.config.js`

Reply:
Fixed, thanks!